src/lib.rs

```diff
@@ -1025,6 +1048,36 @@ where
             ),
         })
     }
+
+    ///
+    /// Creates a request builder for obtaining metadata about a previously received token.
+    ///
+    /// See https://tools.ietf.org/html/rfc7662
+    ///
+    pub fn introspect<'a>(
+        &'a self,
+        token: &'a AccessToken,
+    ) -> Result<IntrospectionRequest<'a, TE, TIR, TT>, ConfigurationError> {
+        self.oauth2_client.introspect(token)
+    }
+
+    ///
+    /// Creates a request builder for revoking a previously received token.
+    ///
+    /// Requires that [`set_revocation_url()`](Self::set_revocation_url()) have already been called to set the
+    /// revocation endpoint URL.
+    ///
+    /// Attempting to submit the generated request without calling [`set_revocation_url()`](Self::set_revocation_url())
+    /// first will result in an error.
```

Comment:
mind copying this comment (replaced with `set_introspection_url`) to `introspect()` above?